tests/periph_gpio_arduino/Makefile

```diff
@@ -0,0 +1,8 @@
+include ../Makefile.tests_common
+
+FEATURES_REQUIRED = periph_gpio
```

Comment:
Hmm, I would have guessed that `FEATURES_REQUIRED += arduino` would be required. But compilation now only fails for the ATmega328p based boards. (Due to lack of ROM/RAM.)

Reply:
the `USEMODULE += arduino` will pull in the `arduino` feature, see `Makefile.dep` - that's how it is handled for other modules too.

However, to me this is not consistent or logically sound in all places, so you got a point there. But also: that's another PR to, if we want to change something there.

Reply:
> the `USEMODULE += arduino` will pull in the `arduino` feature

Aha! Thanks for the explanation :-)